lib/3.6/bundles.cf

```diff
@@ -416,3 +416,54 @@ bundle agent run_ifdefined(namespace, mybundle)
     verbose_mode::
       "$(this.bundle): found matching bundles $(bundlesfound) for namespace '$(namespace)' and bundle '$(mybundle)'";
 }
+bundle agent sysctl(settings_array)
```

Comment:
I think we might be very likely to conflict with someone elses bundle here. Perhaps call the bundle cfe_stdlib_sysctl